tests/clients/test_google-cloud-storage.py

```diff
@@ -14,7 +14,7 @@
 from opentelemetry.trace import SpanKind
 
 from mock import patch, Mock
-from six.moves import http_client
+import http.client as http_client
```

Comment:
Following your Slack poll result:

```suggestion
from http import client as http_client
```

Reply:
My bad, forgot to commit this change after the poll.